app/models/host/discovered.rb

```diff
@@ -105,6 +104,7 @@ def populate_fields_from_facts(*params)
     end
 
     populate_discovery_fields_from_facts(facts)
+    create_notification
```

Comment:
I moved the creation of the notification *after* facts are populated, so that we can know the Host organization to fetch the org admins. If we do this prior to importing the facts, we would have to guess the organization - replicating the logic that already does that at fact-import time.